### ISSUE TITLE
[Request[ history view - lack of proper operation description

### ISSUE THREAD

**Issue by KptChaos** [jwilk/mbank-cli#103]
I'm not sure, but I think they've messed something up in the new history view. One of the major problems for me is that when you view the account history, you don't see the personal details of the person who made the transfer - in case that person used Express Elixir. In the history you will only see "EXPRESS ELIXIR TRANSFER RECEIVED" without any personal details like name, address etc. To get them, you have to click on the transfer in question in the list and select - "Details". I don't know if this can be fixed by replacing this useless piece of information with these "details".

**Comment by jwilk** [comment]
This is only about express transfers, right?
(For me, non-express transfers show up fine.)

Are both `history` and `history2019` commands affected?

**Comment by KptChaos** [comment]
> It's only about express transfers, right?

Exactly, but if you get them often it's frustrating because you have to log into your account to check who transferred the money. I sent a note to mbank support, but I doubt they'll do anything about it.

**Comment by jwilk** [comment]
I never receive any express transfers, so I have no means to test this myself.

So, uh, if anyone from Poland wants to help by making an express transfer to me, please drop me an e-mail. (There are no guaranteed returns for this investment.)

**Comment by KptChaos** [comment]
I'm willing to help :)
